app/questionnaire/rules.py

```diff
@@ -8,18 +8,34 @@
 logger = logging.getLogger(__name__)
 
 
+def evaluate_comparison_rule(when, answer_value, comparison_value):
+    """
```

Comment:
Minor point . Perhaps a name such as evaluate_answer_comparison_rule may be more explicit since evaluate_comparison_rule could mean comparing answers to constants ?

Reply:
Not averse to changing, but I think it could be used in both cases?